### ISSUE TITLE
Downloading stops after a couple of downloads

### ISSUE THREAD

**Issue by zdavatz** [Flimm/backup-all-my-flickr-photos#11]
After downloading a couple of images, I get the following error:

```
Item 0006/72995 entitled
-10649600% of -1 bytesTraceback (most recent call last):
  File "/usr/bin/backup-all-my-flickr-photos", line 4, in <module>
    __import__('pkg_resources').run_script('backup-all-my-flickr-photos==0.7', 'backup-all-my-flickr-photos')
  File "/usr/lib64/python2.7/site-packages/pkg_resources/__init__.py", line 744, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib64/python2.7/site-packages/pkg_resources/__init__.py", line 1506, in run_script
    exec(script_code, namespace, namespace)
  File "/usr/lib64/python2.7/site-packages/backup_all_my_flickr_photos-0.7-py2.7.egg/EGG-INFO/scripts/backup-all-my-flickr-photos", line 239, in <module>

  File "/usr/lib64/python2.7/site-packages/backup_all_my_flickr_photos-0.7-py2.7.egg/EGG-INFO/scripts/backup-all-my-flickr-photos", line 49, in main

  File "/usr/lib64/python2.7/site-packages/backup_all_my_flickr_photos-0.7-py2.7.egg/EGG-INFO/scripts/backup-all-my-flickr-photos", line 225, in automatic_retry

  File "/usr/lib64/python2.7/site-packages/backup_all_my_flickr_photos-0.7-py2.7.egg/EGG-INFO/scripts/backup-all-my-flickr-photos", line 49, in <lambda>

  File "/usr/lib64/python2.7/site-packages/backup_all_my_flickr_photos-0.7-py2.7.egg/EGG-INFO/scripts/backup-all-my-flickr-photos", line 156, in download_item

  File "/usr/lib64/python2.7/site-packages/backup_all_my_flickr_photos-0.7-py2.7.egg/EGG-INFO/scripts/backup-all-my-flickr-photos", line 198, in guess_extension

  File "/usr/lib64/python2.7/rfc822.py", line 393, in __getitem__
    return self.dict[name.lower()]
KeyError: 'content-disposition'
```

**Comment by Flimm** [comment]
The new version 0.8 should fix this, thanks to the pull request #5.

**Comment by zdavatz** [comment]
Ok, will test asap.
